executor-scripts/linux/link

```diff
@@ -34,6 +34,13 @@ is_vlan() {
 case "$PHASE" in
 pre-up)
 	if [ "${IF_LINK_TYPE}" = "dummy" ]; then
+		if [ -d "/sys/class/net/${IFACE}" ]; then
+			iface_type=$(ip -d link show dev "${IFACE}" | head -n3 | tail -n1 | awk '{ print $1 }')
```

Comment:
I am a few years too late, but the Busybox based `ip` included in the base Alpine doesn't support the `-d` flag here, causing an `ifup` for dummy interfaces that already exist to fail. Installing `iproute2-minimal` resolves this.

Is there another way we could detect if the existing interface is of the correct type? I just encountered an issue that I tracked back to this, so thought it worth commenting in case others encounter it.

Reply:
Hm, we could dump the check for the interface type and just check if an interface with that name exists instead?

@kaniini  any thoughts?